src/nvim/ex_cmds2.c

```diff
@@ -744,40 +744,42 @@ void dbg_breakpoint(char_u *name, linenr_T lnum)
 
 static char_u   *profile_fname = NULL;
 
-/*
- * ":profile cmd args"
- */
+/// ":profile cmd args"
+///
+/// @param eap Pointer to the command arguments structure.
 void ex_profile(exarg_T *eap)
 {
-  static proftime_T pause_time;
+  static proftime_T  pause_time;
+  int                len;
+  char_u            *arg;
```

Comment:
The legacyVim code follows that convention (and must, because of C89), but we do not.